src/fides/api/ctl/routes/util.py

```diff
@@ -80,3 +92,20 @@ async def forbid_if_editing_any_is_default(
                 != existing_resources[resource["fides_key"]].is_default
             ):
                 raise errors.ForbiddenError(sql_model.__name__, resource["fides_key"])
+
+
+# Map the ctl model type to the scope prefix.
+# Policies and datasets have ctl-* prefixes to
+# avoid overlapping with ops scopes of same name
+CLI_SCOPE_PREFIX_MAPPING: Dict[str, str] = {
+    "data_category": DATA_CATEGORY,
+    "data_qualifier": DATA_QUALIFIER,
+    "data_subject": DATA_SUBJECT,
+    "data_use": DATA_USE,
+    "dataset": CTL_DATASET,
+    "evaluation": EVALUATION,
+    "organization": ORGANIZATION,
+    "policy": CTL_POLICY,
+    "registry": REGISTRY,
+    "system": SYSTEM,
+}
```

Comment:
Used for dynamically adding the new scopes to the crud endpoints

Reply:
Thanks for highlighting! I missed this first pass.